exchanges/okx/okx.go

```diff
@@ -942,23 +671,23 @@ func (ok *Okx) GetAlgoOrderList(ctx context.Context, orderType, algoOrderID, cli
 	if limit > 0 {
 		params.Set("limit", strconv.FormatInt(limit, 10))
 	}
-	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getAlgoOrderListEPL, http.MethodGet, common.EncodeURLValues(getAlgoOrders, params), nil, &resp, true)
+	var resp []AlgoOrderResponse
+	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getAlgoOrderListEPL, http.MethodGet, common.EncodeURLValues("trade/orders-algo-pending", params), nil, &resp, request.AuthenticatedRequest)
 }
 
 // GetAlgoOrderHistory load a list of all algo orders under the current account in the last 3 months.
 func (ok *Okx) GetAlgoOrderHistory(ctx context.Context, orderType, state, algoOrderID, instrumentType, instrumentID string, after, before time.Time, limit int64) ([]AlgoOrderResponse, error) {
-	params := url.Values{}
 	if orderType == "" {
-		return nil, errors.New("order type is required")
+		return nil, order.ErrTypeIsInvalid
```

Comment:
Needs test coverage, will stop reporting them from here but please re-review the offline ones we can at least test